src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -241,17 +243,25 @@ export default class SpecialResolutionForm extends Mixins(DateMixin) {
   get getCreateResolutionResource (): any {
     return this.getResource.changeData?.specialSpecialResolution?.sampleFormSection || {}
   }
-
+  /** download URL fot pdf file */
   get documentURL (): string {
-    const docUrl = sessionStorage.getItem('BASE_URL') +
+    /**
+     * In session is stored the BASE_URL with business ID
+     * Here to acccess asset folder, need to remove business ID from URL
+     */
+    const baseUrlwithId = sessionStorage.getItem('BASE_URL')
+    const businessId = sessionStorage.getItem('BUSINESS_ID')
+
+    const baseUrlwithoutId = baseUrlwithId.replace(new RegExp(`${businessId}/?`), '')
```

Comment:
is there any other way to get only base URL without business-id?
in session storage, it is set along with business, Need to access the files folder in public folder, 
so remove business id from base URL and using it.
Please let me know is there any other way to access

Reply:
Why do you need to access files in the public folder? Or, why can't you do this the same way that Create UI does it?

Reply:
Users want to download PDF, which is located in public/file folder. I create UI  we are using only BASEURL, but session it store only BASE_URL (without BusinessId), in edit UI, we are store BASE_URL with business ID
create UI
![Screen Shot 2022-08-10 at 2 00 33 PM](https://user-images.githubusercontent.com/55506249/184019147-3d8e6289-0b65-4bc7-a215-3b30190b0322.png)
In edit UI
![Screen Shot 2022-08-10 at 2 01 00 PM](https://user-images.githubusercontent.com/55506249/184019234-7f821051-998f-46a1-9bb7-544845373ed0.png)

Reply:
Here is why the Edit UI base URL includes the business id.

https://github.com/bcgov/business-edit-ui/blob/6071429797646b3a9296414bf4da61fd040cb24a/src/utils/fetchConfig.ts#L133

Maybe you can use `process.env.BASE_URL` directly as per this code.

Reply:
updated